<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_day">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>75</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>DAY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_night">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>75</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>NIGHT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
